src/applications/ivc-champva/10-7959C/config/constants.js

```diff
@@ -18,3 +20,33 @@ export const applicantListSchema = (requireds, propertyList) => {
     },
   };
 };
+
+// These proptypes are used with the custom pages that use
+// ApplicantRelationshipPage
+export const pageProps = {
+  data: PropTypes.object,
+  goBack: PropTypes.func,
+  goForward: PropTypes.func,
+  pagePerItemIndex: PropTypes.string || PropTypes.number,
+  setFormData: PropTypes.func,
+  updatePage: PropTypes.func,
+  onReviewPage: PropTypes.bool,
+};
+export const reviewPageProps = {
+  data: PropTypes.object,
+  editPage: PropTypes.func,
+  pagePerItemIndex: PropTypes.number,
+  props: PropTypes.object,
+  title: PropTypes.func,
+};
+
+export const yesNoOptions = [
+  {
+    label: 'Yes',
+    value: 'yes',
+  },
+  {
+    label: 'No',
+    value: 'no',
+  },
+];
```

Comment:
These constants are used in any custom page leveraging `ApplicantRelationshipPage`